vessim/__init__.py

```diff
@@ -3,7 +3,7 @@
 import vessim.analysis
 import vessim.core
 import vessim.cosim
-import vessim.sil
+import vessim.sil  # noqa: F401
```

Comment:
unless obvious, let's document such noqa's a bit. F401 is "[Module imported but unused](https://www.flake8rules.com/rules/F401.html)", why is that the case?